utils.sh

```diff
@@ -278,7 +271,8 @@ _is_terminal() {
 # Globals: None
 # Arguments: 2
 #   ${1} - value of field to fetch from json
-#   ${2} - Optional, nth number of value from extracted values, default it 1.
+#   ${2} - Optional, no of lines to parse for the given field in 1st arg
```

Comment:
First, this change doesn't affect anything, i have updated the rest of the code to function in the same way it was before.

In almost all of the api requests, we just need to fetch only value, even if more are available.

So, just decreasing the load before send the output to sed command.

Now, say in case we need to access all values, well that's also done.

you can see the same done in https://github.com/Akianonymus/google-drive-upload/blob/fix/upload.sh#L1155 line.

Lastly, it doesn't involve any extra command and we can obviously control the whole output, and also doesn't increase the load, so i don't see any problem with this.